mozilla_bitbar_devicepool/test_run_manager.py

```diff
@@ -165,7 +165,13 @@ def handle_queue(self, project_name, projects_config):
     def thread_active_jobs(self):
         while self.state == 'RUNNING':
             logger.info('getting active runs')
-            self.process_active_runs()
+            try:
+                self.process_active_runs()
+            except requests.exceptions.ConnectionError as e:
+                logger.warning("exception raised when calling process_active_runs.")
+                logger.warning(e)
+                # TODO: if we see this a lot, add exponential backoff?
+                time.sleep(10)
```

Comment:
Seems a little redundant with the other sleep but it is ok.

Reply:
We sleep a bit longer on exceptions (20s vs 10s). More time for the service to recover.